app/javascript/components/search/results/ResultsPanel.js

```diff
@@ -86,10 +86,11 @@ const FacetResultsFooter = ({ studySearchState }) => {
           <FontAwesomeIcon icon={faInfoCircle} className="fa-lg fa-fw icon-left"/>
         </div>
         <div className="">
-          <p>Our advanced search is metadata-powered.  By selecting filters, your search <b>targets only studies that use ontology terms</b> in their metadata file.
-          Currently, about 20% of public studies supply that metadata.</p>
-          Learn more about our search capability on our <a href="https://github.com/broadinstitute/single_cell_portal/wiki/Search-Studies" target="_blank" rel="noreferrer">wiki</a>.<br/>
-          Study authors looking to make their studies more accessible can read our <a href="https://github.com/broadinstitute/single_cell_portal/wiki/Metadata-File#Metadata-powered-Advanced-Search" target="_blank" rel="noreferrer">metadata guide</a>.
+          <p>By selecting filters, your search targets <b>only studies that use ontology terms</b> in their metadata file.
+          <br/>
+          Currently, about 20% of public studies supply such metadata.</p>
```

Comment:
I've heard "70 of 300" (23%), but I'm not sure when that was queried.  If it was a few months ago, I'd expect it to be 25-30% now, and worth updating given some hesitance about the low figure.

Reply:
I'm not crazy about having `70 out of ~300` hard-coded in the UI.  This will only get more and more stale the longer it is up.  As of 10AM on 4.12.2021, it is 72 out of 335, but by the time this is released, that will likely be higher for both.

I think we should either update these numbers dynamically (like the home banner does), or simply give an approximate percentage range, like 20-30%, or "roughly one quarter", or "only a subset of public studies".